ftw/raven/config.py

```diff
@@ -32,6 +33,36 @@ def ignored_exception_classnames(self):
 
         return tuple(ignored)
 
+    @property
+    def tags(self):
+        value = self._get_tags_from_file()
+        value.update(self._get_tags_from_env())
```

Comment:
Reading this, it looks like the values from file and env are actually *merged*, with the values specified directly in the env variable taking precedence. That wasn't entirely clear from the README, I would have expected the file to be completely ignore if there are values in the env variable :wink: